tiledb/__init__.py

```diff
@@ -2,6 +2,8 @@
 import os
 import sys
 
+from tiledb.libtiledb import version as libtiledb_version
```

Comment:
Let's avoid adding this to the public API - if we have to do this to fix imports we should import as `__libtiledb_version`

Reply:
I hadn't thought of it, but I think I've found a good solution!